c-api/dotnet/src/Picky/Picky.csproj

```diff
@@ -0,0 +1,37 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Company>Devolutions</Company>
+    <Description>Bindings to Rust picky native library</Description>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Did you check this is good for DVLS? Note that you can target multiple frameworks if needed (`<TargetFrameworks>{comma-seperated list of frameworks}</TargetFrameworks>`)

Reply:
DVLS is still using the .NET Framework so we'd need at least to support the old and the new .NET runtime in the project file

Reply:
@CBenoit See [here](https://docs.microsoft.com/en-us/dotnet/standard/frameworks)